migration/iD12/oracle/202411301658_IDEMPIERE-6326.sql

```diff
@@ -0,0 +1,33 @@
+-- IDEMPIERE-6326 Rename "Automatic Commit" user preference to "Save on Navigate"
+SELECT register_migration_script('202411301658_IDEMPIERE-6326.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Nov 30, 2024, 5:03:05 PM MYT
+UPDATE AD_Element SET Description='Automatically save changes to the current record upon user navigation.', Help='Automatically save changes to the current record when the user navigates to another record, switches to a detail tab, or performs an action (e.g., running a process) that requires saving the current changes.',Updated=TO_TIMESTAMP('2024-11-30 17:03:05','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Element_ID=202879
```

Comment:
Hi @hengsin you didn't change the Name on AD_Element, so the next Synchronize Terminology set back the name to "AutoCommit".

Also, there is an AD_Message (recordId 155) with the AutoCommit message, I think is not used anywhere, but maybe is good also to synchronize it.